README.md

```diff
@@ -0,0 +1,44 @@
+# example-approver-policy-plugin
+
+This repo contains an example [cert-manager/approver-policy plugin](https://cert-manager.io/docs/projects/approver-policy/#plugins).
+
+
+> :warning:  This is plugin is not meant to be actually used and does not contain a best practices production ready code.
```

Comment:
`to actually be used`

`and its code is neither production ready nor does it follow best practices`